tests/hipscat_import/test_run_catalog_import.py

```diff
@@ -0,0 +1,106 @@
+"""Functional tests for catalog import"""
+
+import os
+
+import hipscat_import.catalog.run_import as runner
+import pytest
+from hipscat.catalog.catalog import Catalog
+from hipscat_import.catalog.arguments import ImportArguments
+from hipscat_import.catalog.file_readers import CsvReader
+
+from hipscat_cloudtests import TempCloudDirectory, assert_parquet_file_ids
+
+
+@pytest.mark.dask
+def test_catalog_import_write_to_cloud(
+    dask_client,
+    small_sky_parts_dir_local,
+    tmp_dir_cloud,
+    example_cloud_storage_options,
+    tmp_path,
+):
+    """Using local CSV files, write a new catalog to the cloud bucket."""
+    with TempCloudDirectory(
```

Comment:
The temp directory is currently `<prefix_path>/<method_name><some random string>`. Can we add some sort of separator between the `method_name` and the `random_string`, e.g. "-"? Otherwise it might be hard to tell them apart if we need to!